EssentialApp/EssentialAppTests/Helpers/ListViewController+TestHelpers.swift

```diff
@@ -42,10 +42,34 @@ extension ListViewController {
 	}
 }
 
+extension ListViewController {
+	func numberOfRenderedComments() -> Int {
+		numberOfRows(in: commentsSection)
+	}
+
+	func commentUsername(at row: Int) -> String? {
+		commentView(at: row)?.usernameLabel.text
+	}
+
+	func commentDate(at row: Int) -> String? {
+		commentView(at: row)?.dateLabel.text
+	}
+
+	func commentMessage(at row: Int) -> String? {
+		commentView(at: row)?.messageLabel.text
+	}
+
+	func commentView(at row: Int) -> ImageCommentCell? {
```

Comment:
This helper is not used outside of this extension, so it should be marked as `private`:
```suggestion
	private func commentView(at row: Int) -> ImageCommentCell? {
```